cypress/e2e/27-opt-in-trial-banner.cy.ts

```diff
@@ -0,0 +1,76 @@
+import { BannerStack, MainSidebar, SigninPage, WorkflowPage } from '../pages';
+import planData from '../fixtures/Plan_data_opt_in_trial.json';
+import { INSTANCE_OWNER, BACKEND_BASE_URL } from '../constants';
+
+const mainSidebar = new MainSidebar();
+const bannerStack = new BannerStack();
+const signinPage = new SigninPage();
+const workflowPage = new WorkflowPage();
+
+describe('BannerStack', { disableAutoLogin: true }, () => {
+	before(() => {
+		const now = new Date();
+		const fiveDaysFromNow = new Date(now.getTime() + 5 * 24 * 60 * 60 * 1000);
```

Comment:
Not very important, but could be a bit cleaner if we use luxon `DateTime` and [`plus` function](https://moment.github.io/luxon/api-docs/index.html#datetimeplus)

Reply:
Did not want to use a dependency only to add 5 days to a date!